tests/runtime/call

```diff
@@ -326,6 +326,13 @@ REQUIRES_FEATURE dpath kfunc
 TIMEOUT 5
 AFTER ./testprogs/syscall read
 
+NAME path
+RUN {{BPFTRACE}} -ve 'kfunc:filp_close { $f = path(args->filp->f_path); if (!strcontains($f, "tmp")) { printf("OK\n"); exit(); } }'
+EXPECT OK
+REQUIRES_FEATURE dpath kfunc
+TIMEOUT 5
+AFTER ./testprogs/syscall read
+
```

Comment:
would also be nice to have a runtime test that just tests with a few combinations of constants to make sure that codepath works

Reply:
Do you have  an example of runtime test？

Reply:
`RUN {{BPFTRACE}} -ve 'BEGIN { if (!strcontains("abc", "a")) { printf("OK\n"); exit(); } }'`